src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePicker.cs

```diff
@@ -115,6 +115,24 @@ public FlyoutPlacementMode FlyoutPlacement
 
 		#endregion
 
+		#region TimePickerFlyoutStyle DependencyProperty
+
+		public Style TimePickerFlyoutStyle
```

Comment:
go with the property name `FlyoutPresenterStyle`, for consistency. Please add a comment too mentioning that this is an Uno-only property to allow styling the FlyoutPresenter.